pavelib/quality.py

```diff
@@ -55,7 +55,11 @@ def find_fixme(options):
     Run pylint on system code, only looking for fixme items.
     """
     num_fixme = 0
-    systems = getattr(options, 'system', '').split(',') or ALL_SYSTEMS
+    systems = getattr(options, 'system', '')
+    if systems:
+        systems = systems.split(',')
+    else:
+        systems = ALL_SYSTEMS
```

Comment:
Now we have this code in two places. Might be easier to put ALL_SYSTEMS back to a string.

Reply:
Yes - I agree. I'll do so.